include/swift/AST/ASTBridgingImpl.h

```diff
@@ -467,7 +467,7 @@ BridgedConformance BridgedSubstitutionMap::getConformance(SwiftInt index) const
 //===----------------------------------------------------------------------===//
 
 swift::Fingerprint BridgedFingerprint::unbridged() const {
-  return swift::Fingerprint({this->v1, this->v2});
+  return swift::Fingerprint(swift::Fingerprint::Core(this->v1, this->v2));
```

Comment:
Does this work with brace initialization of the type?

```c++
swift::Fingerprint::Core{this->v1, this->v2}
```

Reply:
Done